CHANGELOG.md

```diff
@@ -1,3 +1,84 @@
+## Fleet 4.62.1 (Jan 13, 2025)
+
+### Bug fixes
+
+* Fixed issue when identical MDM commands were sent twice to the same device when replica DB was being used.
```

Comment:
anything other than this should have already been merged... looking for the rogue pr